.github/workflows/build.yml

```diff
@@ -25,6 +25,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     continue-on-error: ${{ matrix.experimental }}
     steps:
+      - uses: step-security/harden-runner@63c24ba6bd7ba022e95695ff85de572c04a18142 # v2.7.0
+        with:
+          egress-policy: audit
```

Comment:
I also want to assess behavior on `master`; hence the suggestion to change the policy only in a subsequent PR.

Reply:
^ Changes my mind; will apply most changes in this PR.